fattmerchant/Merchant.py

```diff
@@ -281,3 +283,387 @@ def set_merchant_id(self, id):
+
+            {
+            "company_name": "Fattmerchant",
+            "contact_name": "Fattmerchant",
+            "contact_email": "info2@fattmerchant.com",
+            "contact_phone": "8555503288",
+            "address_1": "25 Wall Street",
+            "address_2": "Suite 1",
+            "address_city": "Orlando",
+            "address_state": "FL",
+            "address_zip": "32801"
+            }
+
+        .. note::
+            Authentication Token and Team Admin Status Required
+
+        """
+        endpoint = u"team"
+        answer = self.request.put_request(endpoint, body=team_info)
+        logger.debug(json.loads(answer))
+        return json.loads(answer)
+
+    def team_edit_branding_info(self, branding_info):
+        u"""somethign
+        
+        .. note::
+            Authentication Token and Team Admin Status Required
+        
+        This resource gives merchant admins the ability to change their company's branding.
+        branding may be uploaded as either a PNG or JPEG image file.
+        A merchant's branding will appear anywhere a merchant's object is returned.
+        The merchant branding will be securely hosted on Amazon S3 bucket and recorded in our database.
+
+        Arguments:
+            branding_info {dict} -- [description]
+
+        example
+
+        .. code-block:: JSON
+
+            {
+                "name": "Mercedes-Benz",
+                "image": "logo.png"
+            }
+
+        """
+        endpoint = u"team/option/branding"
+        answer = self.request.put_request(endpoint, body=branding_info)
+        logger.debug(json.loads(answer))
+        return json.loads(answer)
+    
+    def team_list_members(self):
+        u"""
+        .. note::
+            Authentication Token and Team Admin Status Required
+        
+        Retrieves all the users on a merchants's team, showing team-oriented details and team_role.
+        Used for finding all members of a team and listing them by roles.
+        """
+        endpoint = u"team/user"
+        answer = self.request.get_request(endpoint)
+        logger.debug(json.loads(answer))
+        return json.loads(answer)
+    
+    def team_create_member(self,user_info):
+        u"""
+        .. note::
+            Authentication Token and Team Admin Status Required
+        
+        This creates a whole new user account in your merchant team.
+        Automatically adds the user to the merchant team.
+        Can only be used by users with team_rule of "admin".
+        This is one of two ways to create a new user account; the other being POST /self.
+        Requires a valid email, password and name at minimum.
+        If team_role is not selected, it will default to "full".
+        Used by team admins who want to add members to their merchant account who do not already have a Fattmerchant account.
+       
+        
+        Arguments:
+            user_info {[type]} -- user information in a dictionary
```

Comment:
Can you go over all the doc strings and make sure they are all correct, this seems to be missing the type of `dict` and I saw a few others that weren't entirely correct. If don't care that much right now about the doc strings that is ok too and we can disregard this for now.